drivers/pinctrl/pinctrl_mchp_xec_pvt.h

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2019 Intel Corporation
+ * Copyright (c) 2022 Microchip Technology Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __MCHP_XEC_PVT_H
```

Comment:
note: no content in this header is shared, so please just put everything in the source file using this content (also, this header is not self-contained)

Reply:
ok, will move header file contents to source file